src/components/Table/index.js

```diff
@@ -14,53 +31,147 @@ const Table = ({ columns, data }) => {
     []
   )
 
-  const { getTableProps, headerGroups, rows, prepareRow } = useTable(
+  const {
+    getTableProps,
+    headerGroups,
+    page,
+    prepareRow,
+    canPreviousPage,
+    canNextPage,
+    pageCount,
+    gotoPage,
+    nextPage,
+    previousPage,
+    state
+  } = useTable(
     {
       columns,
       data,
-      defaultColumn
+      manualPagination: true,
+      defaultColumn,
+      initialState: { pageIndex, pageSize },
+      pageCount: Math.ceil(total / pageSize)
     },
-    useFlexLayout
+    useFlexLayout,
+    usePagination
   )
 
+  useEffect(() => {
+    const { pageSize, pageIndex } = state
+    onChange({ pageSize, pageIndex })
+  }, [onChange, state])
+
   return (
-    <div {...getTableProps()} className={tableStyle}>
-      {/* TABLE HEAD */}
-      <div>
-        {headerGroups.map((headerGroup, i) => (
-          <div key={i} {...headerGroup.getHeaderGroupProps({})} className={th}>
-            {headerGroup.headers.map((column, j) => (
-              <div key={j} {...column.getHeaderProps()}>
-                {column.render("Header")}
+    <>
+      <pre>
```

Comment:
This needs to be removed before we merge it. If you like it you can add a boolean `debug` property to the table and print this only when `debug` is true